libs/geospatial/src/main/java/org/codice/ddf/libs/geo/util/GeospatialUtil.java

```diff
@@ -35,6 +35,8 @@
 
   public static final String EPSG_4326_URN = "urn:ogc:def:crs:EPSG::4326";
 
+  public static final String EPSG_X_4326_URN = "urn:x-ogc:def:crs:EPSG:4326";
```

Comment:
Is this being used?

Reply:
https://github.com/glenhein/ddf/blob/DDF-3552/catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v1_1_0/catalog/source/WfsFilterDelegate.java#L117